feat(zod-openapi): Make context accessible in the doc route

Karibash opened the pull request:
close: #243

yusukebe commented:
Hi @Karibash 

I'm sorry to be late.

Looks good! One thing. Could you write the short usage of this feature on README? Thanks!

Karibash commented:
@yusukebe 
I wasn't sure where to add the explanation, but I added it to the Tips section anyway.
Also, I found some typos, so I fixed them as well.

yusukebe commented:
Hi @Karibash 

That place is good!

Let's ship it now. Thanks!
